cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/distributed/AbstractDistributedProgramRunner.java

```diff
@@ -164,6 +169,50 @@ public TwillController launch(TwillApplication twillApplication) {
   }
 
   /**
+   * Gets plugin files that needs to be localized for the adapter. If the given run is not an adapter, no
+   * modification will be done to the map.
+   */
+  private Map<String, File> addAdapterPluginFiles(ProgramOptions options, Map<String, File> localizeFiles) {
```

Comment:
Why needs to be a noun? This method is to add files that needs to be localized to the given map. Shouldn't it call "addXXX" ?